SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2156,9 +2166,46 @@ bool USpatialNetDriver::FindAndDispatchStartupOpsServer(const TArray<Worker_OpLi
 		}
 	}
 
+	auto IsVirtualWorkerTranslatorReady = [this]()
+	{
+		return VirtualWorkerTranslator != nullptr ? VirtualWorkerTranslator->IsReady() : true;
```

Comment:
Nit: a bit unintuitive, that `VirtualWorkerTranslator == nullptr` would return true.

Reply:
Agreed.